drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -481,10 +481,12 @@ static int gpio_ite_get_config(const struct device *dev,
 	reg_1p8v = &IT8XXX2_GPIO_GCRX(
 			gpio_1p8v[gpio_config->index][pin].offset);
 	mask_1p8v = gpio_1p8v[gpio_config->index][pin].mask_1p8v;
```

Comment:
mask_1p8v is only used in the condition.  No need to assign it to a value if it is not used.  Move it to inside the new register exclusion condition.

Reply:
Done